### PR TITLE
Styled all the pages that the user will see

### PR DESCRIPTION
**Pull request by lisa-sa-li**
## Steps to QA

- Create a superuser account
- Just scroll, click, resize the page as much as you can to see if things look ugly

### CODE REVIEW

File: hackathon_site/event/jinja2/event/base.html [hunk 1]
```diff
@@ -11,14 +11,21 @@
         <!-- Compiled and minified CSS for Materialize -->
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" />
 
-        <title>{% block title %}Hackathon Site{% endblock %}</title>
+        <!-- Google Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700&amp;display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Nunito:400,600,700&amp;display=swap" rel="stylesheet">
+        
+        <title>{% block title %}MakeUofT 2021{% endblock %}</title>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/base.html]: Note for the template repo - should we make this from the `HACKATHON_NAME` and `EVENT_START_DATE` settings, or make a note to change it when customizing? Either one could be done in https://github.com/ieeeuoft/hackathon-template/pull/238

File: hackathon_site/event/static/event/styles/scss/styles.scss [hunk 2]
```diff
@@ -20,21 +20,42 @@ h3 {
...
+// Background color change
+.wrapper {
+    transition: background-color 0.3s ease-out;
+}
+
+.scrollColor {
+    transition: color 0.3s ease-out;
+}
+
+.ombreBackground {
```
Comment (grahamhoyes) [inline hackathon_site/event/static/event/styles/scss/styles.scss:49]: I don't think these colors are actually ombre?

File: hackathon_site/event/jinja2/event/reset_password/reset_password_confirm.html [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {% extends "event/form_base.html" %}
 
+{% block title %}Reset Password - MakeUofT 2021{% endblock %}
 {% block nav_logout_login_links %}{% endblock %}
 
 {% block form_title %}{{ "Invalid Link" if validlink==False else "Reset Password" }}{% endblock %}
-{% block form_img %}<img src="{{ static('event/images/password.svg') }}" alt="Reset password image"/>{% endblock %}
+{% block form_img %}<object type="image/svg+xml" data="{{ static('event/images/password.svg') }}" alt="Change password image"></object>{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/reset_password/reset_password_confirm.html]: alt should be "Reset password image"?

File: hackathon_site/event/jinja2/event/reset_password/reset_password_complete.html [hunk 4]
```diff
@@ -1,19 +1,22 @@
 {% extends "event/base.html" %}
 
+{% block title %}Reset Password - MakeUofT 2021{% endblock %}
 {% block nav_logout_login_links %}{% endblock %}
 
-{% block form_img %}<img src="{{ static('event/images/password.svg') }}" alt="Reset password image"/>{% endblock %}
+{% block form_img %}<object type="image/svg+xml" data="{{ static('event/images/password.svg') }}" alt="Change password image"></object>{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/reset_password/reset_password_complete.html]: ditto about alt attribute?

File: hackathon_site/event/jinja2/event/reset_password/reset_password_confirm.html (hunk 3 above)
Comment (grahamhoyes) [reply hackathon_site/event/jinja2/event/reset_password/reset_password_confirm.html]: Actually, it looks like `object` doesn't take an `alt` attribute: https://www.w3schools.com/tags/tag_object.asp

File: hackathon_site/event/jinja2/event/reset_password/reset_password.html [hunk 5]
```diff
@@ -1,11 +1,12 @@
 {% extends "event/form_base.html" %}
 
+{% block title %}Reset Password - MakeUofT 2021{% endblock %}
 {% block form_title %}Password Reset{% endblock %}
 
 {% block nav_logout_login_links %}{% endblock %}
 
 {% block form_description %}<p class="col s12" style="margin-bottom: 10px;">Forgot your password? Enter your email address below and we’ll email instructions for setting a new one.</p>{% endblock %}
-{% block form_img %}<img src="{{ static('event/images/password.svg') }}" alt="Reset password image"/>{% endblock %}
+{% block form_img %}<object type="image/svg+xml" data="{{ static('event/images/password.svg') }}" alt="Change password image"></object>{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/reset_password/reset_password.html]: Not going to comment on the rest of them, but alt attribute again

File: hackathon_site/event/jinja2/event/landing.html [hunk 6]
```diff
@@ -12,299 +14,365 @@
...
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                Will you be shipping out hardware?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Unfortunately, our team would not be able to ship out any hardware for MakeUofT 2021. But the first 200 people to apply will be able to get 25 dollars reimbursement after submitting a demo with their team!
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html]: "would not" -> "will not"
"first 200 people to apply" -> "first 200 people to be accepted"

File: hackathon_site/event/jinja2/event/landing.html [hunk 7]
```diff
@@ -12,299 +14,365 @@
...
+
+                <div class="col s12 m6">
+                    <ul class="collapsible">
+                        <li>
+                            <div class="collapsible-header">
+                                How does MakeUofT work?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    This year MakeUofT has changed its format to accommodate the difficulties of hosting a virtual makeathon. For MakeUofT 2021, instead of a traditional makeathon, you get to spend up to 2 weeks on the project you are working on and submit it! The challenges for MakeUofT would be announced during the event’s opening ceremony on February 6th, 2021.
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html]: "The challenges for MakeUofT would be announced" -> "will be announced"

File: hackathon_site/event/jinja2/event/landing.html [hunk 8]
```diff
@@ -12,299 +14,365 @@
...
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                Who can apply? When is the deadline to apply?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    We’re welcoming any students of age 18 or over pursuing post-secondary education! Applications open on January 7th, 2021 and close at 11:59 PM EST on February 4th. Acceptances will be internally decided on a rolling basis, so apply early to maximize your chances of getting accepted!
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:233]: According to the code, they open January 6 and close February 5 at 11:59. If logistics wants it to be the 7th-4th, we should update the settings file

File: hackathon_site/event/jinja2/event/landing.html [hunk 9]
```diff
@@ -12,299 +14,365 @@
...
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                Do I have to order my own hardware? Will reimbursements be available?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    The first 200 people to apply (and submit a demo) would be able to get 25 dollars reimbursement!!
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html]: "would" -> "will"
"first 200 people to apply" -> "first 200 people to be accepted"

Also the two exclamation marks seems a bit tacky to me... it's only $25

File: hackathon_site/event/jinja2/event/landing.html [hunk 10]
```diff
@@ -12,299 +14,365 @@
...
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                What size of team is allowed?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Feel free to make a team with up to 4 members!
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html]: Do we have a minimum size?

File: hackathon_site/event/jinja2/event/landing.html [hunk 11]
```diff
@@ -12,299 +14,365 @@
...
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                Will reimbursements be available?
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html]: This is already a question above, "Do I have to order my own hardware? Will reimbursements be available?"

File: hackathon_site/event/jinja2/event/landing.html [hunk 12]
```diff
@@ -12,299 +14,365 @@
...
+            <h3 class="center">Check out last year!</h3>
+            <iframe class="video center" src="https://www.youtube.com/embed/KPeTfGlh9DE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
         </div>
+        <br /><br />
+    </div>
+    <br /><br /><br />
+
+    <div class="carousel">
+        <!-- IMAGES MUST ALL BE THE SAME SIZE -->
+        <a class="carousel-item" ><img src="https://i.imgur.com/u9JEte7.jpg" alt="2020-image-1"/></a>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:314]: These make the page load really slowly, can we shrink them all down to 500px height or smaller?

File: hackathon_site/event/jinja2/event/landing.html [hunk 13]
```diff
@@ -12,299 +14,365 @@
...
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
+                <img src="{{ static('event/images/sponsors.svg') }}" alt="One big sponsor image"/>
                 <br />
             </div>
             <p class="center">
                 If you would like to sponsor us, feel free to reach out:
-                <a href="mailto:hello@email.com" class="hoverLink">hello@email.com</a>
+                <a href=mailto:{{ contact_email }}" class="hoverLink">{{ contact_email }}</a>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:344]: Created an issue to update this in the template repo: https://github.com/ieeeuoft/hackathon-template/issues/240

File: hackathon_site/event/jinja2/event/landing.html [hunk 14]
```diff
@@ -12,299 +14,365 @@
...
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                When can I expect to hear back?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Acceptances would be sent out on a rolling basis, so apply early!
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html]: would -> will

File: hackathon_site/event/jinja2/event/dashboard_base.html [hunk 15]
```diff
@@ -1,194 +1,192 @@
...
-            <p class="faqQuestion">Other questions?</p>
-            <p class="faqAnswer">If you have any questions or concerns, feel free to contact us at <a href="mailto:{{ contact_email }}" class="primaryText hoverLink">{{ contact_email }}</a>.</p>
+            <div class="borderTopDiv z-depth-1">
+                <h2 class="formH2" id="faq">Application FAQs</h2>
+
+                <p class="faqQuestion">When can I expect to hear back?</p>
+                <p class="faqAnswer">Acceptances will be sent out on a rolling basis, so apply early, and you will hear back sooner!</p>
+                
+                <p class="faqQuestion">Do I need to be in a team? What can I do if I don’t have a team?</p>
+                <p class="faqAnswer">No, you don’t have to be in a team! In fact, we only accept individual applications from applicants. However, feel free to form a team with other accepted applicants and collaborate on the same project.</p>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/dashboard_base.html]: Just to be clear, are we allowing people to submit individual projects? Or is this question only about being in a team for applying.

File: hackathon_site/event/jinja2/event/login.html [hunk 16]
```diff
@@ -1,10 +1,11 @@
 {% extends "event/form_base.html" %}
 
+{% block title %}Login - MakeUofT 2021{% endblock %}
 {% block form_title %}Login{% endblock %}
 
 {% block nav_logout_login_links %}{% endblock %}
 
-{% block form_img %}<img src="{{ static('event/images/login.svg') }}" alt="Login image"/>{% endblock %}
+{% block form_img %}<object type="image/svg+xml" data="{{ static('event/images/login.svg') }}" alt="Login image"></object>{% endblock %}
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/login.html]: The object tag doesn't appear to make images scale like `img` did, so on mobile this doesn't shrink down:

image

Same with the rest of the images I assume

File: hackathon_site/event/jinja2/event/landing.html (hunk 8 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html:233]: I'll ask them

File: hackathon_site/event/jinja2/event/landing.html (hunk 10 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html]: I dont think so but I will ask to confirm

File: hackathon_site/event/static/event/styles/scss/styles.scss (hunk 2 above)
Comment (lisa-sa-li) [reply hackathon_site/event/static/event/styles/scss/styles.scss:49]: ??

File: hackathon_site/event/jinja2/event/landing.html (hunk 8 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html:233]: "Jan 7th to Feb 4th would be better for us in order to for us to deal with the last minute applications"

File: hackathon_site/event/jinja2/event/landing.html (hunk 10 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html]: teams of 1 are fine

File: hackathon_site/event/jinja2/event/dashboard_base.html (hunk 15 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/dashboard_base.html]: yes, if they are in a team of 1

Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/dashboard_base.html]: I made in more clear

File: hackathon_site/event/jinja2/event/login.html (hunk 16 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/login.html]: fixed in styles.css

File: hackathon_site/event/jinja2/event/base.html (hunk 1 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/base.html]: ya we should

File: hackathon_site/event/jinja2/event/landing.html (hunk 12 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html:314]: done

File: hackathon_site/event/static/event/styles/scss/styles.scss (hunk 2 above)
Comment (grahamhoyes) [reply hackathon_site/event/static/event/styles/scss/styles.scss:49]: Isn't ombre a brownish color?

File: hackathon_site/event/jinja2/event/landing.html (hunk 8 above)
Comment (grahamhoyes) [reply hackathon_site/event/jinja2/event/landing.html:233]: Sounds good, can you update the settings file to match?

File: hackathon_site/event/jinja2/event/dashboard_base.html (hunk 15 above)
Comment (grahamhoyes) [reply hackathon_site/event/jinja2/event/dashboard_base.html]: Looks good!

File: hackathon_site/event/jinja2/event/landing.html (hunk 8 above)
Comment (grahamhoyes) [reply hackathon_site/event/jinja2/event/landing.html:233]: Scratch that, you already did!

**Review by grahamhoyes — DISMISSED** [review]
Looks good overall to me.

There's a bit of padding to the left of the hamburger menu button on mobile, not sure if that was already there in the template. I'll re-approve if you want to fix that 
image

File: hackathon_site/event/static/event/styles/scss/styles.scss (hunk 2 above)
Comment (grahamhoyes) [reply hackathon_site/event/static/event/styles/scss/styles.scss:49]: TIL it's french for "shaded"

**Review by grahamhoyes — APPROVED** [review]
LGTM

**Pull request merged**